src/Core/Manifest/ClassManifest.php

```diff
@@ -195,6 +93,35 @@ public function __construct($base, $includeTests = false, $forceRegen = false, $
         }
     }
 
+    public function getParser()
+    {
+        if (!$this->parser) {
+            $this->parser = (new ParserFactory)->create(ParserFactory::PREFER_PHP5);
```

Comment:
Any reason for `PREFER_PHP5` over 7? [PHP-Parser docs](https://github.com/nikic/PHP-Parser/blob/master/doc/2_Usage_of_basic_components.markdown#parsing) say:

> Unless you have strong reason to use something else, `PREFER_PHP7` is a reasonable default.

The PHP 7 grammar looks to have slightly fewer rules, is it possibly marginally more performant that way around?